app/views/viewlets/charm-details.js

```diff
@@ -60,36 +61,17 @@ YUI.add('charm-details-view', function(Y) {
         this.charmView.setAttrs(cfg);
       } else {
         container.delegate('click', this.close, '.close-slot', this);
-        panel.removeClass('animate-in');
-        container.hide();
-
-        store.charm(charm.get('storeId'), {
```

Comment:
This is not necessary because the `BrowserCharmView` handles fetching the data if it's not passed in.